lib/pbench/server/api/resources/query_apis/__init__.py

```diff
@@ -577,8 +575,8 @@ def expect_index(self, dataset: Dataset) -> bool:
         ignore the requirement (which is to be sure we don't strand the
         Elasticsearch documents).
 
-        There are several factors that might tell use not to expect an index
-        map to appear:
+        We don't expect an index map (and therefore can proceed with a bulk
+        index operation) if:
```

Comment:
The concept of doing a bulk index operation is beyond the scope of this function.  I would replace the parenthetical with `(and therefore we return 'False')`.